demos/examples/UnbiasedCompetitionPNAS2019_tutorial/NWepanechnikovKernelRegrRaster.m

```diff
@@ -0,0 +1,99 @@
+function [inst_rate,t] = NWepanechnikovKernelRegrRaster(time,raster,pop,kwidth,Ts,flag_interp)
+
+%  Nadaraya-Watson smoothing kernel regression of a rastergram using a Epanechnikov kernel
+
+  if nargin < 6
+    flag_interp = 0; % set this to 1 if you want to interpolate to the previous sampling frequency 1/(dsfact*dt)
+  end
+
+  if isempty(pop)
+    pop = unique(raster(:,2));
+  end
```

Comment:
You could consider using more recent [function argument validation](https://es.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html#:~:text=A%20validation%20function%20is%20a%20MATLAB%20function%20that,the%20value%20of%20the%20arguments%20they%20are%20validating) syntax